src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -91,7 +91,7 @@ define(function (require) {
           var tileLayer = L.tileLayer(tileUrl, {
             attribution: marked(configFile.tilemap_attribution),
             subdomains: configFile.tilemap_subdomains,
-            minZoom: configFile.tilemap_min_zoom,
+            minZoom: Math.max(1, configFile.tilemap_min_zoom),
```

Comment:
Since we're specifically mentioning this in the yml comment, let's do this conversion inside server/config/index.js instead so it only ever needs to happen once.